platform/o.n.swing.laf.flatlaf/src/org/netbeans/swing/laf/flatlaf/fontscolors/FlatLafDark-annotations.xml

```diff
@@ -21,148 +21,132 @@
-    <fontcolor name="org-netbeans-modules-subversion-remote-Annotation"/>
-    <fontcolor name="org-netbeans-modules-tomcat5-error" bgColor="ffff8080"/>
-    <fontcolor name="org-netbeans-modules-versioning-annotate-Annotation"/>
-    <fontcolor name="org-netbeans-modules-web-beans-annotations-decorated-bean"/>
-    <fontcolor name="org-netbeans-modules-web-beans-annotations-delegate-point"/>
-    <fontcolor name="org-netbeans-modules-web-beans-annotations-event"/>
-    <fontcolor name="org-netbeans-modules-web-beans-annotations-injection-point"/>
-    <fontcolor name="org-netbeans-modules-web-beans-annotations-observer"/>
-    <fontcolor name="org-netbeans-modules-web-core-syntax-JspParserErrorAnnotation" waveUnderlined="red"/>
-    <fontcolor name="org-netbeans-modules-xml-error" bgColor="ffffa0a0"/>
-    <fontcolor name="org-netbeans-spi-editor-hints-parser_annotation_err"/>
-    <fontcolor name="org-netbeans-spi-editor-hints-parser_annotation_err_fixable"/>
-    <fontcolor name="org-netbeans-spi-editor-hints-parser_annotation_hint"/>
-    <fontcolor name="org-netbeans-spi-editor-hints-parser_annotation_hint_fixable"/>
-    <fontcolor name="org-netbeans-spi-editor-hints-parser_annotation_todo" foreColor="ffa8c023"/>
-    <fontcolor name="org-netbeans-spi-editor-hints-parser_annotation_todo_fixable" foreColor="ffa8c023"/>
-    <fontcolor name="org-netbeans-spi-editor-hints-parser_annotation_verifier"/>
-    <fontcolor name="org-netbeans-spi-editor-hints-parser_annotation_verifier_fixable"/>
-    <fontcolor name="org-netbeans-spi-editor-hints-parser_annotation_warn"/>
-    <fontcolor name="org-netbeans-spi-editor-hints-parser_annotation_warn_fixable"/>
-    <fontcolor name="OtherThread"/>
-    <fontcolor name="OtherThread_BP" bgColor="ff4B1919"/>
-    <fontcolor name="OtherThread_BP_broken" bgColor="ff283C26"/>
-    <fontcolor name="OtherThread_DBP" bgColor="ff372B2B"/>
-    <fontcolor name="OtherThread_PC" bgColor="ff283C26"/>
-    <fontcolor name="OtherThread_PC_BP" bgColor="ff283C26"/>
-    <fontcolor name="OtherThreads"/>
-    <fontcolor name="OtherThreads_BP" bgColor="ff4B1919"/>
-    <fontcolor name="OtherThreads_BP_broken" bgColor="ff283C26"/>
-    <fontcolor name="OtherThreads_DBP" bgColor="ff372B2B"/>
-    <fontcolor name="OtherThreads_PC_BP" bgColor="ff283C26"/>
-    <fontcolor name="PHPError" waveUnderlined="red"/>
-    <fontcolor name="PHPNotice" waveUnderlined="ffc0c000"/>
-    <fontcolor name="PHPWarning" waveUnderlined="ffc0c000"/>
-    <fontcolor name="resetResultsProfilingPoint" bgColor="ff508098"/>
-    <fontcolor name="resetResultsProfilingPointD" bgColor="ff2B2B41"/>
-    <fontcolor name="stopwatchProfilingPoint" bgColor="ff508098"/>
-    <fontcolor name="stopwatchProfilingPointD" bgColor="ff2B2B41"/>
-    <fontcolor name="takeSnapshotProfilingPoint" bgColor="ff508098"/>
-    <fontcolor name="takeSnapshotProfilingPointD" bgColor="ff2B2B41"/>
+  <fontcolor name="Breakpoint_broken" bgColor="darkGray"/>
```

Comment:
Changing indentation is not nice. Especially when everything else is using 4 spaces in this module.

Reply:
The "ff" alpha prefix is not required. If you use the 1.2 DTD and define color aliases for the used colors then if would be easier to create something similar to FlatLafLight theme, but that's just a recommendation.

Reply:
Fixed indentation.

The aim of my previous and this PR was to port as initial step the NB Darcula color profile as is, conversion to 1.2 DTD with use of color palette and creation of a specific color profile for FlatLaf Light LAF should be done in separate PR in my opinion.